site2/website-next/release-notes/versioned/pulsar-2.10.2.md

```diff
@@ -0,0 +1,220 @@
+- [fix][broker] Fix potential exception causing the reader to be unable to continue reading [#17556](https://github.com/apache/pulsar/pull/17556)
+- [fix][broker] Fix pulsarLedgerIdGenerator can't delete index path when zk metadata store config rootPath. [#17192](https://github.com/apache/pulsar/pull/17192)
+- [fix][broker] Fix schema does not replicate successfully [#17049](https://github.com/apache/pulsar/pull/17049)
+- [fix][broker] Fix the replicator unnecessary get schema request for BYTES schema [#17523](https://github.com/apache/pulsar/pull/17523)
+- [fix][broker] Fixed error when delayed messages trackers state grows to >1.5GB [#16490](https://github.com/apache/pulsar/pull/16490)
+- [fix][broker] Increment topic stats outbound message counters and update rate after messages have been written to the TCP/IP connection [#17043](https://github.com/apache/pulsar/pull/17043)
+- [fix][broker] ManagedCursor: mark delete no callback when create meta-ledger fail [#16841](https://github.com/apache/pulsar/pull/16841)
+- [fix][broker] Multiple consumer dispatcher stuck when `unackedMessages` greater than `maxUnackedMessages` [#17483](https://github.com/apache/pulsar/pull/17483)
+- [fix][broker] PulsarLedgerManager to pass correct error code to BK client [#16857](https://github.com/apache/pulsar/pull/16857)
+- [fix][broker] Release the entry in getEarliestMessagePublishTimeOfPos. [#16386](https://github.com/apache/pulsar/pull/16386)
+- [fix][broker] Retry to delete the namespace if new topics created during the namespace deletion [#16676](https://github.com/apache/pulsar/pull/16676)
+- [fix][broker] Retry when DistributedIdGenerator has BadVersion error [#16491](https://github.com/apache/pulsar/pull/16491)
+- [fix][broker] Skip reading more entries for a pending read with no more entries [#16400](https://github.com/apache/pulsar/pull/16400)
+- [fix][broker] Support loadBalancerSheddingIntervalMinutes dynamic configuration [#16408](https://github.com/apache/pulsar/pull/16408)
+- [fix][broker] The configuration loadBalancerNamespaceMaximumBundles is invalid [#16552](https://github.com/apache/pulsar/pull/16552)
+- [fix][broker] Topic policy reader can't recover when get any exception. [#17562](https://github.com/apache/pulsar/pull/17562)
+- [fix][broker] Unregister topic policy listener if managed ledger close failed [#17652](https://github.com/apache/pulsar/pull/17652)
+- [fix][broker] Upgrade log4j2 version to 2.18.0 [#16884](https://github.com/apache/pulsar/pull/16884)
+- [fix][broker] fix No such ledger exception [#16420](https://github.com/apache/pulsar/pull/16420)
+- [fix][broker] fix broker unackmessages become a negative number [#17003](https://github.com/apache/pulsar/pull/17003)
+- [fix][broker] remove exception log when access status.html [#17025](https://github.com/apache/pulsar/pull/17025)
+- [fix][broker] Consumer can't consume messages because there has two sames topics in one broker [#17526](https://github.com/apache/pulsar/pull/17526)
```

Comment:
```suggestion
- [fix][broker] Consumer can't consume messages because there are two same topics in one broker [#17526](https://github.com/apache/pulsar/pull/17526)
```